core/src/main/python/wlsdeploy/tool/create/domain_creator.py

```diff
@@ -272,6 +272,11 @@ def __run_rcu(self):
         rcu_sys_pass = rcu_db_info.get_preferred_sys_pass()
         rcu_schema_pass = rcu_db_info.get_preferred_schema_pass()
 
+        databaseType = rcu_db_info.get_database_type()
+        if databaseType is not None and databaseType not in ['SSL', 'ATP', 'ORACLE']:
```

Comment:
Please make this list a constant in case we ever need to use it elsewhere and/or change the items in the list.